specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2020-06-01/costmanagement.json

```diff
@@ -1256,7 +1256,7 @@
             "$ref": "#/parameters/scopeQueryParameter"
           },
           {
-            "$ref": "#/parameters/apiVersionParameter"
+            "$ref": "#/parameters/apiVersion20200601Parameter"
```

Comment:
Just curious, why we need put the version info in the description? :) if so, it means we need to add one more definition for every new API version for it?